lib/cloud-metadata/lambda.js

```diff
@@ -0,0 +1,13 @@
+function getMetadataAwsLambda (serviceName) {
+  return {
+    provider: 'aws',
+    region: process.env.AWS_REGION,
+    service: {
+      name: serviceName
+    }
```

Comment:
Good question, and yes, that's one of the fields.  @AlexanderWert [has a RFC going](https://github.com/elastic/ecs/pull/1518) with the Elastic Common Schema (ESC) folks about the new fields getting in/out.

Also -- the `origin` fields are meant to track data that's related to the service that _invoked_ the AWS Lambda function. (is the lambda processing an API Gateway request, or an SQS queue message, or an SNS message, etc.) 

This creates a bit of confusion because 

1. Cloud metadata is (traditionally) part of the client metadata, which is meant to be sent once and only once per request to APM Server
2. An individual Lambda function can be processed invoked by _multiple_ AWS services
3. So it's not clear if this is really where this field belongs or not